src/gr11review/part1/Review1.java

```diff
@@ -1 +1,61 @@
 package gr11review.part1;
+
+import java.io.*;
+
+public class Review1{
+
+    public static void main(String[] args) throws IOException{
+        BufferedReader month = new BufferedReader(new InputStreamReader(System.in));
+        BufferedReader day = new BufferedReader(new InputStreamReader(System.in));
+
+
+        int intMonth; 
+        int intDay;
+        int intDays;
+
+        System.out.print("Enter the month number: ");
+        intMonth = Integer.parseInt(month.readLine());
+
+        System.out.print("Enter the day number: ");
+        intDay = Integer.parseInt(day.readLine());
+
+        if(intMonth == 1){
+            intDays = intDay;
+            System.out.print(intDays);
+        }else if(intMonth == 2){
+            intDays = intDay + 31;
+            System.out.print(intDays);
+        }else if(intMonth == 3){
+            intDays = intDay + 59;
```

Comment:
The logic might work, but is there a way to do this that doesn't require you to manually set these constants in the code (e.g. 31, 59, 90, etc.)? It is fair to code that January = 31, February = 28, March = 31, etc. but there is a more efficient way to do this than running constants all the way up. For example, what if you had to modify this code to account for a leap year? Then you'd be changing 11 months worth of constants.